3-networks/README.md

```diff
@@ -45,13 +48,16 @@ The purpose of this step is to :
 1. Change into 3-networks folder.
 1. Run `cp ../build/tf-wrapper.sh .`
 1. Run `chmod 755 ./tf-wrapper.sh`
-1. Rename terraform.example.tfvars to terraform.tfvars and update the file with values from your environment and bootstrap.
+1. Rename common.auto.example.tfvars to common.auto.tfvars and update the file with values from your environment and bootstrap.
+1. Rename shared.auto.example.tfvars to shared.auto.tfvars and update the file with the target_name_server_addresses.
+1. Rename access_context.auto.example.tfvars to access_context.auto.tfvars and update the file with the access_context_manager_policy_id.
 1. Update backend.tf with your bucket from bootstrap. You can run
 ```for i in `find -name 'backend.tf'`; do sed -i 's/UPDATE_ME/<YOUR-BUCKET-NAME>/' $i; done```.
 You can run `terraform output gcs_bucket_tfstate` in the 0-bootstap folder to obtain the bucket name.
 
 We will now deploy each of our environments(dev/prod/nonprod) using this script.
-When using Cloud Build or Jenkins as your CI/CD tool each environment corresponds to a branch is the repository for 3-networks step and only the corresponding environment is applied.
+When using Cloud Build or Jenkins as your CI/CD tool each environment corresponds to a branch is the repository for 3-networks step
+and only the corresponding environment is applied.
```

Comment:
is there a reason to brake this line? @amandakarina 

and instead of 

> corresponds to a branch **is** the repository for 3-networks

it should be 

> corresponds to a branch **in** the repository for 3-networks

Reply:
It's a looong line. But I can keep long...